testing/test_skipping.py

```diff
@@ -235,6 +235,20 @@ def test_func2():
             ["*def test_func():*", "*assert 0*", "*1 failed*1 pass*"]
         )
 
+    def test_xfail_run_with_skip_mark(self, testdir):
```

Comment:
you can [parametrize](https://docs.pytest.org/en/stable/parametrize.html) the test here to make it cleaner